src/components/core/Dropdown/Dropdown.tsx

```diff
@@ -10,9 +10,10 @@ type Props = {
   onClose: () => void;
   children?: ReactNode;
   position: 'right' | 'left';
+  style?: React.CSSProperties;
```

Comment:
maybe instead of passing in a custom `style`, we update `position` to have another `full-width` option to compete with `left` and `right`.

I imagine this won't be the last time we want a full width version of the dropdown